cmd/janitor/gcp_janitor.py

```diff
@@ -283,7 +288,9 @@ def collect(project, zones, age, resource, filt, clear_all):
         'list',
         '--format=json(name,creationTimestamp.date(tz=UTC),createTime.date(tz=UTC),zone,region,isManaged)',
         '--project=%s' % project])
-    if resource.condition == 'zone' and resource.name != 'sole-tenancy' and resource.name != 'network-endpoint-groups':
+    if (resource.condition == 'zone'
```

Comment:
What's the reason for deleting `resource.name != 'network-endpoint-groups'`? Was it by mistake?

Reply:
Great catch, thanks! Wanted to move per-resource match to separate lines, lost during reformatting.